docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3.3"
+
+services:
+  db:
+    image: postgis/postgis
+    environment:
+      - POSTGRES_DB=ais_dev
+      - POSTGRES_USER=ais_dev
+      - POSTGRES_PASSWORD=ais_dev
+    volumes: 
+      # - './volume/:/var/lib/postgresql/data' #Uncomment for persistent database storage
+      - './init.sql:/docker-entrypoint-initdb.d/init.sql'
+
+  ais:
+    build: .
+    environment: 
+      - DJANGO_SETTINGS_MODULE=ais.local.settings
+    command: python manage.py runserver 0.0.0.0:8080 --settings local_settings
+    volumes:
+      - .:/ais
+    ports:
+      - "8080:8080"
+    depends_on:
+      - db
```

Comment:
Revert the commit of this file as Docker is depended on PR #673, make sure to now commit this file.  
Remove it from the PR, but you can have the change locally.